screen/wallets/entropyGenerator.js

```diff
@@ -0,0 +1,426 @@
+      } else if (selectedIndex === 1) {
+        // btc was selected
+        // index 1 radio - segwit single address
+        w = new SegwitP2SHWallet();
+        w.setLabel(label || loc.wallets.details_title);
+      } else {
+        // btc was selected
+        // index 2 radio - hd bip84
+        if (isTestMode) {
+          w = new HDSegwitBech32Wallet({ network: bitcoin.networks.testnet });
+        } else {
+          w = new HDSegwitBech32Wallet();
+        }
+        w.setLabel(label || loc.wallets.details_title);
+      }
+      if (selectedWalletType === ButtonSelected.ONCHAIN) {
+        await w.generateMnemonicFromEntropy(shuffledEntropy);
+        addWallet(w);
+        await saveToDisk();
+        A(A.ENUM.CREATED_WALLET);
+        ReactNativeHapticFeedback.trigger('notificationSuccess', { ignoreAndroidSystemSettings: false });
+        if (w.type === HDSegwitP2SHWallet.type || w.type === HDSegwitBech32Wallet.type) {
+          navigate('PleaseBackup', {
+            walletID: w.getID(),
+          });
+        } else {
+          goBack();
+        }
+      }
+    } else if (selectedWalletType === ButtonSelected.VAULT) {
+      setIsLoading(false);
+      navigate('WalletsAddMultisig', { walletLabel: label.trim().length > 0 ? label : loc.multisig.default_label });
+    } else if (selectedWalletType === ButtonSelected.LDK) {
+      setIsLoading(false);
+      createLightningLdkWallet(w);
+    }
+  };
+
+  const createLightningLdkWallet = async (wallet) => {
+    const foundLdk = wallets.find((w) => w.type === LightningLdkWallet.type);
+    if (foundLdk) {
+      return alert('LDK wallet already exists');
+    }
+    setIsLoading(true);
+    wallet = new LightningLdkWallet();
+    wallet.setLabel(label || loc.wallets.details_title);
+
+    await wallet.generate();
+    await wallet.init();
+    setIsLoading(false);
+    addWallet(wallet);
+    await saveToDisk();
+
+    A(A.ENUM.CREATED_WALLET);
+    ReactNativeHapticFeedback.trigger('notificationSuccess', { ignoreAndroidSystemSettings: false });
+    navigate('PleaseBackupLdk', {
+      walletID: wallet.getID(),
+    });
+  };
+
+  const createLightningWallet = async (wallet) => {
+    wallet = new LightningCustodianWallet();
+    wallet.setLabel(label || loc.wallets.details_title);
+
+    try {
+      const lndhub = walletBaseURI?.trim();
+      if (lndhub) {
+        const isValidNodeAddress = await LightningCustodianWallet.isValidNodeAddress(lndhub);
+        if (isValidNodeAddress) {
+          wallet.setBaseURI(lndhub);
+          await wallet.init();
+        } else {
+          throw new Error('The provided node address is not valid LNDHub node.');
+        }
+      }
+      await wallet.createAccount(isTestMode);
+      await wallet.authorize();
+    } catch (Err) {
+      setIsLoading(false);
+      console.warn('lnd create failure', Err);
+      return alert(Err.message || Err);
+      // giving app, not adding anything
```

Comment:
giving up?

Reply:
@muursh  this one is an old comment from previous developers , I have remove this now